src/compas_rhino/artists/mixins/edgeartist.py

```diff
@@ -90,9 +90,10 @@ def draw_edges(self, keys=None, color=None):
                                        normalize=False)
         lines = []
         for u, v in keys:
+            start, end = self.datastructure.edge_coordinates(u, v)
             lines.append({
-                'start': self.datastructure.vertex_coordinates(u),
-                'end': self.datastructure.vertex_coordinates(v),
+                'start': start,
+                'end': end,
                 'color': colordict[(u, v)],
                 'name': self.datastructure.edge_name(u, v),
```

Comment:
would you mind fixing this as well?
the `edge_name` method does not exits anymore. see above...

Reply:
No problem!